### ISSUE TITLE
Virtual thumbs up/thumbs down during aciton item

### ISSUE THREAD

**Issue by ddipanfilo** [stride-nyc/remote_retro#282]
As a participant, I want to be able to say whether I'd like to continue or stop talking about a topic once the facilitator prompts me to vote thumbs up or down during the action item stage.  This might also require the facilitator to be able to clear all users thumbs up/thumbs down after they have voted.

This will allow teams to decide if they should continue discussing a topic.

**Comment by vanderhoop** [comment]
hey @ddipanfilo, quick clarification. you're saying that, in your in-person retros, you do this thumps up/down practice to decide whether to continue discussing? if so, how many folks need to say that, yes, they'd like to keep discussing? the majority of participants?

**Comment by ddipanfilo** [comment]
yes, the decision to proceed to a new topic or continue discussing the current topic is based on a majority vote
